src/activities/GenerateQueryWhereClause.ts

```diff
@@ -54,18 +57,22 @@ export default class GenerateWhereClause implements IActivityHandler {
         if (!queryFields) {
             throw new Error("queryFields is required");
         }
+        
 
-        let where = whereClause && whereClause.length > 0 ? whereClause : "1=1";
+        let where = whereClause && whereClause.length > 0 ? whereClause : "";
         const definitionExpression = getDefinitionExpression(layer);
         if (definitionExpression) {
-            where = `${where} AND ${definitionExpression}`;
+            where = `${definitionExpression} AND ${where}`;
```

Comment:
I think we need `(${definitionExpression}) AND (${where})` to avoid unintended effects of OR statements in either part.